<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User</class>
 <widget class="QWidget" name="User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0a0e14;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>40</pointsize>
       <bold>true</bold>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4fc3f7;
        padding-bottom: 15px;
        border-bottom: 2px solid #1e1e1e;
        margin-bottom: 30px;</string>
     </property>
     <property name="text">
      <string>USER CONFIGURATION</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hostnameLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;</string>
     </property>
     <property name="text">
      <string>Hostname:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="hostnameEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QLineEdit {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
          selection-background-color: #4fc3f7;
          selection-color: #000000;
        }
      </string>
     </property>
     <property name="placeholderText">
      <string>Enter system hostname</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="usernameLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QLineEdit {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
          selection-background-color: #4fc3f7;
          selection-color: #000000;
        }
      </string>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
     <property name="placeholderText">
      <string>Enter username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="passwordLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QLineEdit {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
          selection-background-color: #4fc3f7;
          selection-color: #000000;
        }
      </string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your user password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="confirmPassLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #64d2ff;</string>
     </property>
     <property name="text">
      <string>Confirm Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmPassEdit">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QLineEdit {
          background-color: #1e1e1e;
          color: #ffffff;
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 8px;
          selection-background-color: #4fc3f7;
          selection-color: #000000;
        }
      </string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Retype the password</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="user_btn">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
       <family>SpaceMono Nerd Font</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
        QPushButton {
          color: #64d2ff;
          background-color: rgba(30, 30, 30, 0.7);
          border: 2px solid #4fc3f7;
          border-radius: 5px;
          padding: 10px 30px;
        }
        QPushButton:hover {
          background-color: rgba(79, 195, 247, 0.2);
        }
        QPushButton:pressed {
          background-color: rgba(79, 195, 247, 0.4);
          border-color: #64d2ff;
        }
      </string>
     </property>
     <property name="text">
      <string>NEXT</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>